README.md

```diff
@@ -50,6 +57,12 @@ Add the following to your POM file:
          <useTestNG>false</useTestNG>
          <!-- The naming scheme to use for the generated test classes.  One of 'simple' or 'feature-title' --> 
          <namingScheme>simple</namingScheme>
+         <!-- The class naming pattern to use.  Only required/used if naming scheme is 'pattern'.-->
+         <namingPattern>Parallel{c}IT</namingPattern>
+         <!-- Generate JUnitReRun runners instead of JUnit and TestNG single runner. --> 
+         <useJUnitReRun>true</useJUnitReRun>
```

Comment:
Is a separate property required for specifying rerun reports?

If retryCount > 0, that indicates that retry is required.